src/utils/fileName.ts

```diff
@@ -0,0 +1,9 @@
+import { extname } from 'path';
+
+export const fileName = (_req, file, callback) => {
+  const uniqueSuffix = Date.now() + '-' + Math.round(Math.random() * 1e9);
+  const ext = extname(file.originalname);
+  const filename = `${uniqueSuffix}${ext}`;
+
+  callback(null, filename);
+};
```

Comment:
look like you took it from internet and its fine, but you still need to add types

Reply:
updated